lib/searchName.ml

```diff
@@ -97,13 +98,7 @@ let search_key_aux aux conf base an =
       | None -> (an, acc)
     else (an, acc)
   in
-  let acc =
-    Mutil.filter_map
-      (fun i ->
-        let p = Util.pget conf base i in
-        if search_reject_p conf base p then None else Some p)
-      acc
-  in
+  let acc = Mutil.filter_map (fun i -> Util.pget_opt conf base i) acc in
```

Comment:
`let acc = Mutil.filter_map (Util.pget_opt conf base) acc in` ?

Reply:
You are certainly right! I am not a real so APL expert, and I tend to explicit the parametres. Altogether, does this impact performances?
Thanks you anyway